# ngs-for-clinicians

work on Mehis Pold idea: an open source tool that facilitates interpretation of a whole-genome NGS genetic finding for clinicians.

See [discussion on linkedin](http://www.linkedin.com/groups/Who-is-up-momentous-task-1902623.S.238138831?view=&gid=1902623&item=ANET%3AS%3A238138831&trk=NUS_DIG_RITM-title) for context.

## motivation

 a lack of computational tools that enable a high-throughput interpretation of whole-genome NGS finding. For example, Illumina can sequence an entire human genome now faster than 24h. If you were to ask, however, anyone who interprets the whole-genome data then I am doubtful you would find anyone who’d be willing to spend just 24 h on data-interpretation when nothing obvious such as a known disease variants matching the patient’s phenotype is found. Hence, this open source project would simply aim to shorten the current gap between the capacities that produce and interpret the whole-genome NGS data. 

 ## short term roadmap

 I would take a top-down approach. Namely, we will first formulate the high-level plan for the version 1 of this tool. For example, I imagine that the version 1 will integrate the capacities that already exist for interpretation of both known and unknown human variants: OMIM, dbSNP, Exome project, COSMIC, Gene-ontology based pathway analysis etc., and produce reports that take into account the ACMG guidelines for reporting and interpreting a genetic finding. I imagine a capacity to interpret the DNA methylation patterns being part of the initial version as well. The version 1 will most certainly be perfect because the NGS-world is still a world where the unknown outweighs the known. Also, more advance features, for example machine-learning based matching of geno- and phenotypes would most certainly require years before it can be meaningfully applied. 

We need to establish close ties with the clinical community in order to learn more about the needs and expectations of the clinical community as well as patients and their parents. Right now, the clinicians, patients/parents, and people who produce whole-genome NGS data live in almost parallel universes. The only way out of this type of existence is a mutual enlightenment. 

I realize the above lacks a lot of necessary detail. Anyhow, everyone who is interested is very welcome to add-on the above. I also see us all asking the questions like ‘who owns the project?’, ‘what are the resources needed?’ etc. I think that the only way to run a project like this is to keep open source and non-profit. The main resource needed at this point is the time from those of you interested in participating.

